resources/compute_virtual_machines.go

```diff
@@ -0,0 +1,608 @@
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("VirtualMachineProperties.HardwareProfile.VMSize"),
+			},
+			{
+				Name:        "additional_capabilities_ultra_s_s_d_enabled",
+				Description: "The flag that enables or disables a capability to have one or more managed data disks with UltraSSD_LRS storage account type on the VM or VMSS Managed disks with storage account type UltraSSD_LRS can be added to a virtual machine or virtual machine scale set only if this property is enabled",
+				Type:        schema.TypeBool,
+				Resolver:    schema.PathResolver("VirtualMachineProperties.AdditionalCapabilities.UltraSSDEnabled"),
+			},
+			{
+				Name:        "computer_name",
+				Description: "Specifies the host OS name of the virtual machine <br><br> This name cannot be updated after the VM is created <br><br> **Max-length (Windows):** 15 characters <br><br> **Max-length (Linux):** 64 characters <br><br> For naming conventions and restrictions see [Azure infrastructure services implementation guidelines](https://docsmicrosoftcom/azure/virtual-machines/virtual-machines-linux-infrastructure-subscription-accounts-guidelines?toc=%2fazure%2fvirtual-machines%2flinux%2ftocjson#1-naming-conventions)",
```

Comment:
remove these the `<br>` I would also think that you can cut the description from them.